pkg/docker/client.js

```diff
@@ -616,12 +616,15 @@
                 });
         };
 
-        this.rmi = function rmi(id) {
+        this.rmi = function rmi(id, forced) {
+            if (forced === undefined)
+                forced = false;
```

Comment:
Calling with `forced || false` is perhaps a bit more succinct, but it's fine like this as well.